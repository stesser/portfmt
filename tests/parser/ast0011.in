PORTSCOUT=	limit:^[0-9.]+$$ # bla
PORTSCOUT=	limit:^[0-9.]+$$	# bla
