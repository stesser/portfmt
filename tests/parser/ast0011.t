PORTSCOUT=	limit:^[0-9.]+$$ # bla
PORTSCOUT=	limit:^[0-9.]+$$	# bla
<<<<<<<<<
{ VARIABLE, line 1, .name = PORTSCOUT, .modifier = =, .comment = # bla, .words = [1]{ limit:^[0-9.]+$$ } }
{ VARIABLE, line 2, .name = PORTSCOUT, .modifier = =, .comment = # bla, .words = [1]{ limit:^[0-9.]+$$ } }
