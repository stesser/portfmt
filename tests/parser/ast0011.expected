{ VARIABLE, line 1, .name = PORTSCOUT, .modifier = =, .comment = # bla, .words = { limit:^[0-9.]+$$ } }
{ VARIABLE, line 2, .name = PORTSCOUT, .modifier = =, .comment = # bla, .words = { limit:^[0-9.]+$$ } }
